README.md

```diff
@@ -158,45 +162,55 @@ Use the [`xrpl.transaction`](https://xrpl-py.readthedocs.io/en/latest/source/xrp
 
 ```py
 from xrpl.models.transactions import Payment
-from xrpl.models.transactions.transaction import Transaction
 from xrpl.transaction import send_reliable_submission
-from xrpl.ledger import get_fee
+from xrpl.ledger import get_latest_validated_ledger_sequence
 
-FEE = get_fee(JSON_RPC_CLIENT)
+current_validated_ledger = get_latest_validated_ledger_sequence(client)
 
 # prepare the transaction
+# the amount is expressed in drops, not XRP
+# see https://xrpl.org/basic-data-types.html#specifying-currency-amounts
 my_tx_payment = Payment(
-    account=TESTNET_CLASSIC_ACCOUNT,
-    amount=_AMOUNT,
-    destination=TESTNET_DESTINATION_ACCOUNT,
-    signing_pub_key=TESTNET_WALLET.pub_key,
-    last_ledger_sequence=TESTNET_WALLET.next_sequence_num + 10,
-    sequence=TESTNET_WALLET.next_sequence_num,
-    fee=FEE,
+    account="rMPUKmzmDWEX1tQhzQ8oGFNfAEhnWNFwz",
+    amount="2200000",
+    destination="rPT1Sjq2YGrBMTttX4GZHjKu9dyfzbpAYe",
+    last_ledger_sequence=current_validated_ledger + 20,
+    sequence=test_wallet.next_sequence_num,
+    fee="10",
 )
 # sign the transaction
-my_tx_payment_signed = safe_sign_transaction(my_tx_payment,TESTNET_WALLET)
+my_tx_payment_signed = safe_sign_transaction(my_tx_payment,test_wallet)
 
 # submit the transaction
-tx_response = send_reliable_submission(my_tx_payment, TESTNET_WALLET, JSON_RPC_CLIENT_TESTNET)
+tx_response = send_reliable_submission(my_tx_payment, test_wallet, client)
+```
+
+#### Get fee from the XRP Ledger
+
+In most cases, you should specify `10` for the `fee` field unless you have a strong reason not to. But if you want to get the [current transaction cost](https://xrpl.org/transaction-cost.html#current-transaction-cost) from the XRP Ledger, you can use the `get_fee` function:
```

Comment:
#197 also adds in autofilling capabilities for transactions

Reply:
```suggestion
In most cases, you can specify the minimum [transaction cost](https://xrpl.org/transaction-cost.html#current-transaction-cost) of `"10"` for the `fee` field. But if you want to get the current load-based transaction cost from the network, you can use the `get_fee` function:
```